### PR TITLE
PWN-9141 - WalletFragment.kt added and logic for navigation changed + home == new `My Crypto` now

### PR DESCRIPTION
**Pull request by kabramovp2p**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-9141

## Description of Work

PWN-9141 - WalletFragment.kt added and logic for navigation changed + home == new `My Crypto` now

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/home/ui/wallet/WalletFragment.kt [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+            ActionButton.TOP_UP_BUTTON -> {
+                presenter.onTopupClicked()
+            }
+            else -> {
+                // unsupported on this screen
+            }
+        }
+    }
+
+    override fun showUserAddress(ellipsizedAddress: String) {
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/home/ui/wallet/WalletFragment.kt:121]: what if you create a ViewState here and just render(viewState) and then call all this methods?

File: app/src/main/res/layout/fragment_wallet.xml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/container"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/bg_smoke">
+
+    <com.google.android.material.appbar.AppBarLayout
+        android:id="@+id/app_bar_layout"
```
Comment (gslevinkov) [inline app/src/main/res/layout/fragment_wallet.xml:12]: appBarLayout

File: app/src/main/res/layout/fragment_wallet.xml [hunk 3]
```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.coordinatorlayout.widget.CoordinatorLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/container"
```
Comment (gslevinkov) [inline app/src/main/res/layout/fragment_wallet.xml:6]: can be named root

File: app/src/main/res/layout/fragment_wallet.xml [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+            <LinearLayout
+                android:id="@+id/containerHeader"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/home_toolbar_height"
+                android:orientation="vertical"
+                app:layout_collapseMode="none">
+
+                <include
+                    android:id="@+id/view_balance"
```
Comment (gslevinkov) [inline app/src/main/res/layout/fragment_wallet.xml:35]: viewBalance / layoutBalance

File: app/src/main/res/layout/fragment_wallet.xml [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="24dp"
+                    android:layout_marginBottom="30dp"
+                    tools:layout_height="50dp" />
+
+            </LinearLayout>
+
+            <include
+                android:id="@+id/layout_toolbar"
```
Comment (gslevinkov) [inline app/src/main/res/layout/fragment_wallet.xml:51]: viewToolbar / layoutToolbar

File: app/src/main/res/layout/fragment_wallet.xml [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+    </com.google.android.material.appbar.AppBarLayout>
+
+    <androidx.swiperefreshlayout.widget.SwipeRefreshLayout
+        android:id="@+id/swipe_refresh_layout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_behavior="com.google.android.material.appbar.AppBarLayout$ScrollingViewBehavior">
+
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/home_recycler_view"
```
Comment (gslevinkov) [inline app/src/main/res/layout/fragment_wallet.xml:68]: recyclerViewHome

File: app/src/main/res/layout/fragment_wallet.xml [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:layout_collapseMode="pin" />
+
+        </com.google.android.material.appbar.CollapsingToolbarLayout>
+
+    </com.google.android.material.appbar.AppBarLayout>
+
+    <androidx.swiperefreshlayout.widget.SwipeRefreshLayout
+        android:id="@+id/swipe_refresh_layout"
```
Comment (gslevinkov) [inline app/src/main/res/layout/fragment_wallet.xml:62]: swipeRefreshLayout

File: app/src/main/java/org/p2p/wallet/home/ui/wallet/WalletFragment.kt (hunk 1 above)
Comment (kabramovp2p) [reply app/src/main/java/org/p2p/wallet/home/ui/wallet/WalletFragment.kt:121]: nihua ne ponyal

File: app/src/main/java/org/p2p/wallet/home/ui/container/MainContainerFragment.kt [hunk 8]
```diff
@@ -184,7 +182,7 @@ class MainContainerFragment :
     override fun navigateFromDeeplink(data: DeeplinkData) {
         when (data.target) {
             DeeplinkTarget.HOME -> {
-                navigate(ScreenTab.HOME_SCREEN)
+                navigate(ScreenTab.MY_CRYPTO_SCREEN)
```
Comment (eduardmaximovich) [inline app/src/main/java/org/p2p/wallet/home/ui/container/MainContainerFragment.kt:185]: maybe we should rename for clarity the DeeplinkTarget.HOME to DeeplinkTarget.MY_CRYPTO ?

**Review by eduardmaximovich — APPROVED** [review]

**Review by gslevinkov — APPROVED** [review]

**Pull request merged**